apis/stardust/api-spec/spec.json

```diff
@@ -487,6 +487,32 @@
         "import { onTakeSnapshot } from '@nebula.js/stardust';\nimport { useState } from '@nebula.js/stardust';\nimport { useLayout } from '@nebula.js/stardust';\n\nconst layout = useLayout();\nconst [zoomed] = useState(layout.isZoomed || false);\n\nonTakeSnapshot((copyOfLayout) => {\n  copyOfLayout.isZoomed = zoomed;\n  return Promise.resolve(copyOfLayout);\n});"
       ]
     },
+    "onContextMenu": {
```

Comment:
Should it be added here ?

Reply:
hmm, I think we said that it should not be added yet. Thats what the "@ignore" should do. Try re-running yarn spec see if it is removed.

Reply:
found it, your ignore was inside the example attribute

Reply:
... so you say it is ok to have it here anyway ?